### ISSUE TITLE
Accessibility Fixes for Tabs Block

### ISSUE THREAD

**Issue by amberhinds** [tomusborne/generateblocks#1081]
### Description
The tabs block is missing the necessary ARIA attributes and to make it accessible for screen reader users.

### Current Code
This is the HTML that I saw for the block when testing.
```
<div class="gb-container gb-container-556f2798 gb-tabs" data-opened-tab="3">
<div class="gb-container gb-container-ef96ba99 gb-tabs__buttons">
   <button class="gb-button gb-button-c9499c31 gb-button-text gb-tabs__button" aria-expanded="false">Tab 1</button>
   <button class="gb-button gb-button-9e60ae75 gb-button-text gb-tabs__button" aria-expanded="false">Tab 2</button>
   <button class="gb-button gb-button-5d733888 gb-button-text gb-tabs__button gb-block-is-current" aria-expanded="true">Tab 3</button>
</div>

<div class="gb-container gb-container-035a7023 gb-tabs__items">
<div class="gb-container gb-container-76170fb9 gb-tabs__item">
<p>Tab 1 content.</p>
</div>

<div class="gb-container gb-container-69add859 gb-tabs__item">
<p>Tab 2 content.</p>
</div>

<div class="gb-container gb-container-50fd7973 gb-tabs__item gb-tabs__item-open">
<p>Tab 3 content.</p>
</div>
</div>
</div>
```

#### Expected behavior
This is a high-level summary of what is missing to make this block accessible:
1. Missing [role="tablist"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tablist_role) on the tabs container.
2. Missing [role="tab"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) on the tabs.
3. Missing [role="tabpanel"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role) on the tab content container.
4. Missing [aria-controls](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls) on the tab buttons to indicate what they control.
5. Tab buttons should be in a list.
6. The use of aria-expanded is incorrect. Tabs should use [aria-selected](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected) instead to indicate which tab is active. Aria-expanded is for multi-selectable elements like accordions, where you can have multiple open or visible at a time. It doesn't really apply to a single tabpanel.

#### References: 
Here are some references to help:
- [Example of Tabs with Manual Activation](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-manual/)
- [Expected Keyboard Interactions for Tabs](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/)
- 

-----

**Comment by Calvin-Susan** [comment]
Thanks for looking into this, @amberhinds 🙌  You are THE BEST!

**Comment by JeanPaiva** [comment]
@amberhinds thank you, We will work to improve this on our next release.

**Comment by tomusborne** [comment]
These improvements have been merged in GBP 1.7.0, which is the next Pro release.

Thanks!

**Comment by Calvin-Susan** [comment]
Thanks for making accessibility improvements a priority, @tomusborne! 💪
